### ISSUE TITLE
Masking in Transformer

### ISSUE THREAD

**Issue by FrancescaDr** [theislab/interscale#43]
### Description of feature

- [ ] #18 
- [ ] Long-range interaction masking: In `mask` from `torch.nn.transformerencoder` set `attn_mask` to `-inf`. Currently, not working because together with `src_padding_mask` it returns NaNs in the global embedding
